Homework7/lesson6/my_weather.py

```diff
@@ -0,0 +1,39 @@
+from requests import get
+import json
+import sys
+
+nargs = len(sys.argv)
+#print(nargs)
+
+if nargs == 1:
+    try:
+        city_name = sys.argv[1]
```

Comment:
Count the amount of time the command `sys.argv[1]` appears in the code. I counted ~3. Try extracting this to a parameter and just use it.

Reply:
It bothered me as well... however, I need to spend time to think how to do that. Tried further reading but nothing productive yet.